sdk/avalon_sdk/ethereum/ethereum_work_order.py

```diff
@@ -220,6 +221,28 @@ def encryption_key_get(self, worker_id, requester_id,
         pass
 
 
+def is_wo_id_in_event(event, *kargs, **kwargs):
```

Comment:
Change function to private

Reply:
It is a static method @Ram-srini , since we are not really dealing with member variables here.